### PR TITLE
Adds resource name and description to launch and schedule prompt wizards

### PR DESCRIPTION
**Pull request by mabashian**
##### SUMMARY
link #8249 
link #7254

Launch with description:
Screen Shot 2021-05-25 at 10 33 08 AM

Launch without description:
Screen Shot 2021-05-25 at 10 33 16 AM

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - UI

### CODE REVIEW

**Review by keithjgrant — APPROVED** [review]

**Comment by nixocio** [comment]
I triggered an e2e test run.

**Comment by tiagodread** [comment]
Hey @mabashian the dev build is rendering the resource name fine, the prod build looks like:
 
image
 
With the resource name max length the user still can fill the wizard ok, but for the description field the user can describe a long 
 description text, because of this, I'd recommend a fixed size for description with scrolling _(textarea)_ option to allow the user fill in the wizard:
  
image

**Comment by mabashian** [comment]
Decided to go with an expand/collapse solution for descriptions longer than 512 characters:

wiz_desc

File: awx/ui_next/src/components/LaunchPrompt/LaunchPrompt.jsx [hunk 1]
```diff
@@ -102,7 +102,24 @@ function PromptModalForm({
           validateStep(nextStep.id);
         }
       }}
-      title={t`Prompts`}
+      title={t`Launch | ${resource.name}`}
+      description={
+        resource.description.length > 512 ? (
+          <ExpandableSection
+            toggleText={
+              showDescription ? t`Hide description` : 'Show description'
```
Comment (nixocio) [inline awx/ui_next/src/components/LaunchPrompt/LaunchPrompt.jsx]:
```suggestion
              showDescription ? t`Hide description` : t`Show description`
```

**Review by nixocio — APPROVED** [review]

File: awx/ui_next/src/components/LaunchPrompt/LaunchPrompt.jsx (hunk 1 above)
Comment (mabashian) [reply awx/ui_next/src/components/LaunchPrompt/LaunchPrompt.jsx]: 👍 good call

**Review by tiagodread — APPROVED** [review]
This looks really good, thanks for adding we can merge once we mark the string to be translated 👍🏽

**Pull request merged**